features/step_definitions/common_steps.rb

```diff
@@ -268,22 +268,26 @@
   attach_file(label, File.join(Dir.pwd, 'fixtures', file))
 end
 
-When /^I click the top\-level summary table Edit link for the section '(.*)'$/ do |field_to_edit|
+When /^I click the top\-level summary table '(.*)' link for the section '(.*)'$/ do |link_name, field_to_edit|
   edit_link = page.find(:xpath, "//h2[contains(text(), '#{field_to_edit}')]/following-sibling::p[1]/a[text()]")
-  edit_link.text().should have_content('Edit')
+  edit_link.text().should have_content(link_name)
   edit_link.click
 end
 
-When /I click the summary table Edit link for '(.*)'$/ do |field_to_edit|
-  edit_link = page.find(:xpath, "//tr/*/span[contains(text(), '#{field_to_edit}')]/../..//a[text()]")
-  edit_link.text().should have_content('Edit')
+When /I click the summary table '(.*)' (link|button) for '(.*)'$/ do |link_name, elem_type, field_to_edit|
+  case elem_type
+    when 'link'
+      edit_link = page.find(:xpath, "//tr/*/span[text() = '#{field_to_edit}']/../..//a[contains(text(), '#{link_name}')]")
+    else
+      edit_link = page.find(:xpath, "//tr/*/span[text() = '#{field_to_edit}']/../..//input[@value = '#{link_name}']")
```

Comment:
Where do we have buttons at the top of a summary table?  That doesn't feel like a pattern we have anywhere?

Reply:
Here, also the copy brief button IIRC

<img width="1032" alt="screen shot 2017-12-06 at 16 54 33" src="https://user-images.githubusercontent.com/3469840/33673976-2abf5312-daa6-11e7-8dd9-20f08ac4b1e3.png">